src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -231,6 +231,7 @@ export type Project = {
 
   // XXX: These are part of the DetailedProject serializer
   dynamicSampling: {
+    next_id: number;
```

Comment:
Is this used?

Reply:
Or is this only for Relay?

Reply:
it's only for relay, but as I get it in the response, I decided to add it here too